docs/limitations.md

```diff
@@ -0,0 +1,72 @@
+- The CPU scraper needs to be enabled to collect the `systm.load.cores` metric, which affects the **Normalized Load** column in the **Hosts** table and the **Normalized Load** visualization on the host detailed view.
+- The [`hostmetrics receiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver) doesn't support CPU and disk metrics on MacOS. These values will stay empty for collectors running on MacOS.
+- The console shows error Log messages when the [`hostmetrics receiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver) can't access some of the process information due to permission issues.
+- The console shows mapping errors initially until mapping occurs.
+
+## Metrics temporal aggregation
+
+The [Elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/elasticsearchexporter) only support histograms with delta temporality,
+as a consequence SDKs must be configured to report them as delta.
+Changing temporal aggregation also impacts [visualizations](#visualizations).
+
+OpenTelemetry metrics data model provides multiple ways to report metrics temporality:
+- cumulative (default)
+- delta preferred
+- low memory
+
+A complete description and examples are provided in [aggregation temporality documentation](https://opentelemetry.io/docs/specs/otel/metrics/supplementary-guidelines/#aggregation-temporality).
+
+The `low memory` is not recommended for consistency because `counter` metrics are reported either as `delta` or
+`cumulative` when they are captured synchronously or asynchronously.
+
+Temporal aggregation effect depends on the OpenTelemetry metric type:
+
+Gauge and up down counters always provide the "last value", which means that the producers of those metrics only reads
+the last value, they don't keep track of the previous nor compute a delta.
+
+| metric type / temporal aggregation | cumulative | delta preferred |
+|------------------------------------|------------|-----------------|
+| gauge                              | last value | last value      |
+| up down counter                    | last value | last value      |
+| counter                            | cumulative | delta           |
+| histogram                          | cumulative | delta           |
+
+
+As a consequence, metrics sent to Elasticsearch currently need to use the "delta preferred" to properly store histograms,
+otherwise they will be discarded by the collector.
+
+Setting `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=delta` should allow to configure SDKs to change the default value.
+(see [reference](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md#environment-variables) on supported SDKs).
+
+In the case were the producer of `counter` or `histogram` metrics can't be configured with `delta preferred` behavior to report them with `delta`
```

Comment:
Histograms is the only reason we are asking for "delta preferred", because we can't store them otherwise.

Doing this change has a side effect all the `counters` visualizations because we don't have a nice way to handle that with a separate setting, but maybe ES|QL support would be enhanced to allow that, but so far we don't have any ETA.

If we already have dashboards that rely on cumulative counters, then we need to not apply this conversion, which means we need to instruct users to apply this conversion to _some_ metrics and not others, which brings another layer of complexity to the end-user.

In a sense, what we need is to apply `cumulativetodeltaprocessor` only for cumulative histograms at the collector level, ideally close to the edge, or even better allow SDKs to apply this only to histograms without changing anything for counters because it would break the visualizations.

I wonder if contributing the ability to set the time aggregation per metric type would be less painful that the path we are trying to take here.